monkey/infection_monkey/telemetry/attack/usage_telem.py

```diff
@@ -0,0 +1,20 @@
+from infection_monkey.telemetry.attack.attack_telem import AttackTelem
+
+
+class UsageTelem(AttackTelem):
+
+    def __init__(self, technique, status, usage):
+        """
+        T1035 telemetry.
+        :param status: ScanStatus of technique
+        :param usage: Usage string
+        """
+        super(UsageTelem, self).__init__(technique, status)
+        self.usage = usage
```

Comment:
Problematic for the same reason I mentioned in T1035. Use enum, not string.
Since you wanted to create a parent class for both telemetries (which is good), you can generalize this as follows:
create an abstract method 'get_msg' which receives no params (except for self), and returns the message of the telem according to the usage enum and status. Implement the function on T1035 and T1129 telemetries.